index.html

```diff
@@ -113,8 +113,8 @@ <h4>EditContext state</h4>
                 <li><dfn>composition start</dfn> which refers to the offset in [=text=] representing the start position of the text being actively composed. The initial value is 0.</p></li>
                 <li><dfn>composition end</dfn> which refers to the offset in [=text=] representing the end position of the text being actively composed. The initial value is 0. [=composition end=] must always be greater than or equal to [=composition start=].</li>
                 <li><dfn>text formats</dfn> which is an array of [=text format=]. The array is initially empty.</li>
-                <li><dfn>control bounds</dfn> is a rectangle describing the area of the screen in which [=text=] is displayed.  It is in the [=client coordinate=] system and the initial x, y, width, and height all 0.</li>
-                <li><dfn>selection bounds</dfn> is the rectangle describing the position of selection. It is in the [=client coordinate=] system and the initial x, y, width, and height are all 0.</li>
+                <li><dfn>control bounds</dfn> is a rectangle describing the area of the screen in which [=text=] is displayed.  It is in the [=client coordinate system=] and the initial x, y, width, and height all 0.</li>
```

Comment:
In both selection and control bounds, should we also mention that the position is relative to the layout viewport and the rectangle is in CSS pixels? Something similar to how [getBoundingClientRect ](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect) is defined.

Reply:
Isn't this implied because they point to the `client coordinate system` definition being added here, which says it uses layout viewport/CSS pixels?

Reply:
Ah I see it says that [here ](https://drafts.csswg.org/cssom-view/#css-pixels)in the spec that the coordinates are in CSS pixels.